src/util/logging.cpp

```diff
@@ -29,6 +28,20 @@ QMutex s_mutexStdErr;
 // The file handle for Mixxx's log file.
 QFile s_logfile;
 
+QLoggingCategory::CategoryFilter oldCategoryFilter = nullptr;
+
+void controllerDebugCategoryFilter(QLoggingCategory* category) {
+    // Configure controller.*.input/output category here, otherwise forward to to default filter.
+    constexpr char controllerPrefix[] = "controller.";
+    const char* categoryName = category->categoryName();
+    if (qstrncmp(categoryName, controllerPrefix, sizeof(controllerPrefix) - 1) == 0) {
+        category->setEnabled(QtDebugMsg, true);
+    } else {
+        oldCategoryFilter(category);
+        category->setEnabled(QtDebugMsg, false);
```

Comment:
I am not sure what this does. Does it mean all other categories are off in case --controllerDebug() is set?
I can deal with both, but I think we should add a comment here.

Reply:
https://github.com/mixxxdj/mixxx/pull/4523/commits/81e5fe313ac2f1b50e24f8e1f120e2cf1a5fd0de